### PR TITLE
Issue#29 New Specification for 2024-03

### PR DESCRIPTION
**Pull request by Budlee**
A new specification for strongly typed relationships

### CODE REVIEW

File: calm/draft/2024-03/meta/core.json [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    "required": [
+      "nodes",
+      "relationships"
+    ]
+  },
+  "defs": {
+    "node": {
+      "type": "object",
+      "properties": {
+        "uniqueId": {
```
Comment (aidanm3341) [inline calm/draft/2024-03/meta/core.json:27]: I noticed that we have a mix of camel case and kebab case in the spec. Most are kebab case but fields like `uniqueId` and `dataClassification` are camel. If there isn't a reason behind this, maybe we could take this opportunity to make them consistent.

Comment (rocketstack-matt) [reply calm/draft/2024-03/meta/core.json:27]: Let's raise that as a separate issue given the amount of changes we've already asked for, but agree with making this happen.

Comment (jpgough-ms) [reply calm/draft/2024-03/meta/core.json:27]: @aidanm3341 agree this should be a separate issue and would be good to do now we're into a next version of the schema.

File: calm/draft/2024-03/meta/core.json [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+          "$ref": "#/defs/authentication"
+        }
+      },
+      "required": [
+        "uniqueId",
+        "relationship-type"
+      ],
+      "additionalProperties": true
+    },
+    "dataClassification": {
```
Comment (jpgough-ms) [inline calm/draft/2024-03/meta/core.json:115]: @aidanm3341 when you create the issue for `unqiue-id` it's worth including `data-classification too`

cc/ @rocketstack-matt

**Review by jpgough-ms — APPROVED** [review]

File: calm/draft/2024-03/meta/core.json (hunk 2 above)
Comment (rocketstack-matt) [reply calm/draft/2024-03/meta/core.json:115]: Already have #45

**Review by aidanm3341 — APPROVED** [review]

**Pull request merged**